test/dat/Hello.avdl

```diff
@@ -16,7 +16,7 @@ protocol Simple {
     FOO,
     BAR, // the bar enum value
     BAZ
-  }
+  } = BAZ
```

Comment:
Rather than replace this enum, can you add a second one with a default? This will guard better against regressions.

Reply:
@mtth , all set! I added a new Enum with the default value case